test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-list-sync-update/README.md

```diff
@@ -0,0 +1,9 @@
+## Description
+
+This test ensures that creation of a multiple target resource created by a ClusterPolicy `generate.cloneList` rule. If it is not generated, the test fails. 
```

Comment:
This is a specific variant of the cloneList capability: it uses a label selector to select resources. This should be mentioned explicitly in the README. This also means we want separate test cases for cloneList which does NOT use a label selector. You don't have to add those to this PR, but just calling out we need them.

Reply:
updated the description and behavior.